### DISCUSSION TITLE
Return value of delete-database

### DISCUSSION THREAD

**Discussion by jsmassa** [replikativ/datahike#599] [General]
So far delete-database could return anything depending on the backend-implementation.

Do we want to leave it this way or ? 

Options:
1. We leave it as is and do not worry at all about the return value.
2. We do it like Datomic and return true as the [documentation](https://docs.datomic.com/client-api/datomic.client.api.html#var-delete-database) says.
3. We return true when the database existed and something happened and false when the database didn't exist.

**Comment by whilo** [comment]
I opt for following Datomic and throwing an error in all other cases.
